enterprise/frontend/src/embedding-sdk/cli/steps/generate-component-files.ts

```diff
@@ -60,9 +60,9 @@ export const generateReactComponentFiles: CliStepMethod = async state => {
     dashboards,
     isNextJs,
 
-    // Enable user switching only when a valid license is present,
-    // as JWT requires a valid license.
-    userSwitcherEnabled: !!token,
+    // Enable user switching only when a valid license is present
+    // as JWT requires a valid license, and using a sample database.
```

Comment:
```suggestion
    // as JWT requires a valid license, and does not use a sample database.
```
You literally have `!` in the condition.